src/query/service/src/servers/http/clickhouse_handler.rs

```diff
@@ -44,7 +44,6 @@ use serde::Serialize;
 use tracing::error;
 use tracing::info;
 
-use crate::interpreters::InterpreterFactory;
```

Comment:
Maybe rename InterpreterFactoryV2 to InterpreterFactory is better?

Reply:
Addressed in https://github.com/datafuselabs/databend/pull/7762/commits/a920a22f5fac34c6ee855286b33eed8a344cc921